### ISSUE TITLE
Support More Port Forwarding Ports

### ISSUE THREAD

**Issue by trymeouteh** [ivpn/desktop-app#192]
# Feature request
Increase the port forwarding port count.

## Description

Allowing your to create one or more ports to be available for port forwarding. This will be useful for users to self host multiple services at once such as self hosting games (Which could take up to 3 ports to do), P2P messaging apps, seed LBRY videos, mine Monero, etc.

On this list below, many VPN providers who offer port forwarding offer more ports than one port.
https://www.reddit.com/r/VPNTorrents/comments/oqnnrq/list_of_vpns_that_allow_portforwarding_2021/

## Describe the solution you'd like

Allowing settings in your IVPN account to have more than one port for port forwarding.

## Describe alternatives you've considered

None

**Comment by ribbybbir** [comment]
I would like port forwarding too. I already got the web hosting semantics down. I could imagine web hosting on a VPN service. I am certain that popular features will potentially increase market sales due to demand met with supply.

However, I also recognize that IVPN is committed to privacy and no-logging policies as primary focus. Therefore, I also opt for researching VPN-based port forwarding of web hosting. Includes hosting game servers as well.

Maybe I got the wrong idea? It's either privacy or just open hosting (although, I am sure that there's a way to mask/spoof IP address identity). Hard to decide.
